### PR TITLE
Add taxonomy service to documentation

### PR DESCRIPTION
**Pull request by dsquirek**

### CODE REVIEW

**Comment by matthuhman** [comment]
:+1:

File: TaxonomyService.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- [Versioning](#versioning)
+
+
+
+#Request Information
+
+
+HTTP method: GET
+Parameters (query/form):
+-        No Parameters
```
Comment (thomasmahoney) [inline TaxonomyService.md]: "No Parameters" isn't true -- we just don't take any _query_ parameters. The example should also be a valid request - what you call an example is more like a demonstration of the format. I would rewrite this like:

HTTP method: GET
Parameters: The only parameter accepted by this service is the name of the desired taxonomy, provided as part of the URL path.

```
https://api.careerbuilder.com/core/taxonomy/{taxonomyName}
```

Example: https://api.careerbuilder.com/core/taxonomy/onet17

File: TaxonomyService.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            }
+        ]
+    }
+}
+```
+
+
+#Response Information
+----------
+The response returns a single data node which contains a list of languages. Each language contains a list of taxonomies within that language.
```
Comment (thomasmahoney) [inline TaxonomyService.md]: This is not completely correct. Lists in JSON start with [, so "data" doesn't contain a list. I would maybe write it like:

The response will contain a single node `data` which maps each language code supported by the taxonomy to a list of the taxonomy's elements for that language.

File: TaxonomyService.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
```
Comment (thomasmahoney) [inline TaxonomyService.md]: also, I think you should move the response description before the example response.

File: TaxonomyService.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- [Available Taxonomies](#available-taxonomies)
+- [Versioning](#versioning)
+
+
+
+#Request Information
+
+
+HTTP method: GET
+Parameters: The only parameter accepted by this service is the name of the desired taxonomy, provided as part of the URL path.
```
Comment (thomasmahoney) [inline TaxonomyService.md]: Let's also add "Taxonomy names are case-sensitive."

File: TaxonomyService.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+https://api.careerbuilder.com/core/taxonomy/{taxonomyName}
+ 
+Example: https://api.careerbuilder.com/core/taxonomy/ ([Desired Taxonomy](#available-taxonomies))/
```
Comment (thomasmahoney) [inline TaxonomyService.md]: You forgot to update this. make it /taxonomy/onet17 or something

File: TaxonomyService.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+            }
+        ]
+    }
+}
+```
+
+
+#Response Information
+----------
+The response returns a single data node which maps each language code supported by the taxonomy to a list of the taxonomy's elements for that language.
```
Comment (thomasmahoney) [inline TaxonomyService.md]: can we move this above the sample response?

**Comment by thomasmahoney** [comment]
:+1:

**Pull request merged**